src/Symfony/Component/Form/Extension/Core/Type/TimeType.php

```diff
@@ -49,6 +51,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         if ('single_text' === $options['widget']) {
             $builder->addViewTransformer(new DateTimeToStringTransformer($options['model_timezone'], $options['view_timezone'], $format));
+
+            // handle seconds ignored by user's browser when with_seconds enabled and seconds is 00
```

Comment:
I think you can leave out ` and seconds is 00` (actually seconds are omitted which you already mentioned).